corehq/ex-submodules/casexml/apps/phone/utils.py

```diff
@@ -72,6 +72,33 @@ def get_or_cache_global_fixture(restore_state, cache_bucket_prefix, fixture_name
     return [data.replace(global_id, b_user_id)] if data else []
 
 
+def get_or_cache_global_fixture_for_case(domain, case_id, cache_bucket_prefix, fixture_name, data_fn):
```

Comment:
Why is it necessary to have a separate function for cases? Since these fixtures are global the results will be identical for all cases and all users right? I think we can just re-use the existing global cache. This will require refactoring the method signature but not too much:
```diff
- def get_or_cache_global_fixture(restore_state, cache_bucket_prefix, fixture_name, data_fn):
+ def get_or_cache_global_fixture(domain, user_id, cache_bucket_prefix, fixture_name, data_fn, overwrite_cache=False):
```

The for cases the `user_id` would be the case ID and we wouldn't need to pass `overwrite_cache`